chains/storytestnet/addresses.yaml

```diff
@@ -0,0 +1,17 @@
+domainRoutingIsmFactory: "0x6f8fdb6275a9309F28348b93A2311B1A21437071"
+interchainAccountIsm: "0x0eAf81268F22e82Ec81f312031126F8DdddfEC25"
+interchainAccountRouter: "0x2f124C7979aE36466b809caA0C3B812aB59B7EA8"
+mailbox: "0xF2657C6e2986754a8351EDDa45629E97263f46D1"
+proxyAdmin: "0xAf47409d9ea3A1EDC9611C89999823844C2d54Be"
+staticAggregationHookFactory: "0x2D58cFecEA1ea0fc8cBa99B6e81d2917E9961a7b"
+staticAggregationIsmFactory: "0xfeaE28b3CfE111932fE6D476fC9567E26D5655B3"
+staticMerkleRootMultisigIsmFactory: "0x5819b8e0eb555DD1d58Bb7866Af5d485a4958F60"
+staticMerkleRootWeightedMultisigIsmFactory: "0x17169Ec27a0b9eA5E49b0322F07bAd3cca7E95d2"
+staticMessageIdMultisigIsmFactory: "0xA782825BF7E64824cB4D668B2D66D0EB3984DEdF"
+staticMessageIdWeightedMultisigIsmFactory: "0x051B57EF8eda07A38D3521C33D2cb3D35Edf548E"
+testRecipient: "0x3544cEe39DFdA00CDd196853b3e62eD010758f93"
+validatorAnnounce: "0xa0d54e08aE5B9386C2EdA1319E13f68aA1D4885B"
+TransparentUpgradeableProxy: "0x4E14A4a64A9eEC00EB773891562BABFE3B824530"
```

Comment:
Hey @atuser14 thank you for your PR to the Hyperlane Registry! Could you please remove the `TransparentUpgradeableProxy` field from this file? After that run `yarn lint --fix` to properly format the file and have the fields in the expected order. 

Finally sync your branch with the main branch to be sure that the ci runs on the latest version of the changes